### ISSUE TITLE
description word on the right side of image

### ISSUE THREAD

**Issue by cyberpunk-edgerunner** [arXiv/html_feedback#3839]
### Description

the word description appears to be on the right side of the image.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

216af2c0-ab4c-4658-b2b1-27e253a29876

### Paper URL

https://arxiv.org/html/2503.06608v1

### Browser

Chrome/136.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
